server/build-plugins.js

```diff
@@ -124,15 +135,35 @@ async function checkForModuleExistence(pkg) {
   return exists && !stat.isSymbolicLink();
 }
 
+// get names of plugins that are available local to the project
+async function getLocalPlugins() {
```

Comment:
I think that this function would be more flexible in general if it was something like `getModules` that took a directory. Then it would be able to load plugins from any directory

Reply:
Yeah, I generally agree with you on this. This function was really just meant as a convenience method to help keep `prepareModules` clean and easier to read though. I'm not sure it would be good to give the impression that local plugins could be retrieved from any other path.